map.conf

```diff
@@ -0,0 +1,12 @@
+author = pauldpickell
+name = UBC Campus
+format = procedural
+spawn_pos_x = 1250
+spawn_pos_y = 1750
+spawn_pos_z = 1250
+schematic_size_x = 2500
```

Comment:
Here and also in other `map.conf` files for `Minetest_Classroom` realm schematics, it would be nice to store some variables that relate to the map projection parameters used to create real world maps. I will open an issue for this and describe there.